libraries/AP_PiccoloCAN/piccolo_protocol/ESCCommands.c

```diff
@@ -0,0 +1,565 @@
+// ESCCommands.c was generated by ProtoGen version 3.2.a
+
+/*
+ * Copyright Currawong Engineering Pty Ltd
+ * www.currawongeng.com
+ * all rights reserved
```

Comment:
just noticed these headers ... "all rights reserved" doesn't make it clear it can be used under GPLv3. Do you mind changing generator to add a GPLv3 compatible line like "may be used under GNU GPLv3 or later" or equivalent? Or change manually?